README.md

```diff
@@ -88,38 +88,28 @@ SurrealDB is an end-to-end cloud-native database designed for modern application
 
 View the [features](https://surrealdb.com/features), the latest [releases](https://surrealdb.com/releases), the product [roadmap](https://surrealdb.com/roadmap), and [documentation](https://surrealdb.com/docs).
 
-<img src="/img/interface.png">
-
-<h2><img height="20" src="/img/contents.svg">&nbsp;&nbsp;Contents</h2>
-
-- [Features](#features)
-- [Documentation](#documentation)
-- [Installation](#installation)
-	- [Install on macOS](#install-on-macos)
-	- [Install on Linux](#install-on-linux)
-	- [Install on Windows](#install-on-windows)
-	- [Run using Docker](#run-using-docker)
-- [Getting started](#getting-started)
-	- [Client side apps](#client-side-apps)
-	- [Server side code](#server-side-code)
-- [Quick look](#quick-look)
-- [Why SurrealDB](#why-surrealdb)
+<!-- <img src="/img/interface.png"> -->
+
+<h2><img height="20" src="./img/contents.svg">&nbsp;&nbsp;Contents</h2>
+
+- [Install on macOS](#install-on-macos)
+- [Install on Linux](#install-on-linux)
+- [Install on Windows](#install-on-windows)
+- [Run using Docker](#run-using-docker)
+		- [Client side apps](#client-side-apps)
+		- [Server side code](#server-side-code)
 	- [Database, API, and permissions](#database-api-and-permissions)
 	- [Tables, documents, and graph](#tables-documents-and-graph)
-	- [Advanced inter-document relations](#advanced-inter-document-relations-and-analysis-no-joins-no-pain)
-	- [Simple schema definition](#simple-schema-definition-for-frontend-and-backend-development)
-	- [Connect directly from web-browsers](#connect-and-query-directly-from-web-browsers-and-client-devices)
-	- [Multiple different query methods](#query-the-database-with-the-tools-you-want)
-	- [Realtime live queries and data changes](#realtime-live-queries-and-data-changes-direct-to-application)
-	- [Scale effortlessly for high-availability](#scale-effortlessly-to-hundreds-of-nodes-for-high-availability-and-scalability)
-	- [Extend your database with JavaScript](#extend-your-database-with-javascript-functions)
-	- [Designed to be embedded or in the cloud](#designed-to-be-embedded-or-to-run-distributed-in-the-cloud)
-- [Community](#community)
-- [Contributing](#contributing)
-- [Security](#security)
-- [License](#license)
```

Comment:
This could be an artifact of `git diff` but it seems like these lines (and many other lines of the table of contents) were deleted and not reintroduced.

Reply:
I'm sorry. I had a plugin installed on visual studio code, that caused formatting error leading to deletion of multiple lines.